MoreLinq/ZipMap.cs

```diff
@@ -0,0 +1,61 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2008 Jonathan Skeet. All rights reserved.
```

Comment:
Notice has incorrect date and author

Reply:
Right. Should be changed to:

```suggestion
// Copyright (c) 2020 Daniel Jonsson. All rights reserved.
```